Classes/PSGitHub.Classes.ps1

```diff
@@ -0,0 +1,257 @@
+Class GitHubPlan {
+    [Int]$Collaborators
+    [String]$Name
+    [Int]$PriavteRepos
+    [Int]$Space
+
+    # The plan object returned from the API call to a Gistcan be passed directly into this constructor.
+    GitHubPlan([Object]$object) {
+        $this.Collaborators = $object.collaborators
+        $this.Name = $object.name
+        $this.PriavteRepos = $object.private_repos
+        $this.Space = $object.space
+    }
+
+    # Empty Constructor.
+    GitHubPlan() { }
+}
+
+Class GitHubUser {
+    [String]$AvatarUrl
```

Comment:
Maybe we could use `System.Uri` instead of "string" for some of these URL-related properties.